### DISCUSSION TITLE
Avast flags VoxelTycoon.exe and quarantines it

### DISCUSSION THREAD

**Discussion by Alastor-git** [voxeltycoon/issues#623] [Troubleshooting] [answered]
**Describe the bug**
Avast flags VoxelTycoon.exe as containing IDP.Generic on the steam beta version and quarantines it/ 

**Steps to reproduce**
Install Avast
Install the Steam beta version
Try to launch the game. 

**Platform**
Windows 10

**Comment by voxeltycoon** [comment, marked as answer]
That's pretty common for Steam games: https://www.google.com/search?q=avast+avg+IDP.Generic+%22steam%22

There's not much we can do with it. We can only advise adding the game to Avast as an exclusion, so their systems could learn that the file is safe and eventually will stop flagging it.
